app/jobs/check_for_bike_serial_searches_job.rb

```diff
@@ -0,0 +1,17 @@
+class CheckForBikeSerialSearchesJob < ApplicationJob
+  queue_as :default
+
+  def perform(serial)
+    bike = Bike.find_by(serial: serial)
```

Comment:
moving this file to the other PR. it makes more sense that it's with the other job